### ISSUE TITLE
naming convention for lensed host galaxies

### ISSUE THREAD

**Issue by mpwiesner** [LSSTDESC/sims_GCRCatSimInterface#89]
Is there a preferred naming convention for lensed host galaxy FITS stamps?  Right now they are named like this: lens_ID+lensed_u_band_magnitude+lensed_g_band_magnitude+lensed_r_band_magnitude+lensed_i_band_magnitude+lensed_z_band_magnitude+lensed_y_band_magnitude_bulge/disk.fits

Example: 9993181_24.528840367986316_24.20642333936395_24.074635430660926_24.02940766644763_24.070640392595934_24.188802301179077_bulge.fits

These names will get put in a list of FITS stamps by the <InstanceCatalogWriter.py> script and somewhere ImSim will have to parse these names.

**Comment by jbkalmbach** [comment]
By lensed_u_band_magnitude do you mean lensed u-band magNorm value? Is it just the u-band magNorm + magnification?

**Comment by mpwiesner** [comment]
@jbkalmbach yes that is correct.  Is this useful to have in the filename?  If not, do you have an idea what filenames should look like?

**Comment by jbkalmbach** [comment]
Phosim Ids are just strings right? So I was anticipating changing DDF lensed object ids from integers to just strings with the form `agn_host_"dc2_sys_id"_"image_number"`. So it would look something like `agn_host_0_0` or `sne_host_10_1`. Because I think we should be able to get those magnitudes out of the truth catalogs right? What do you think?
